recipes/cpprate/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cpprate" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tmaklin/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9734f5e9da27404ad12e5d426d0547d0b35e91b86b37f9f34367570fb5660af4
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make
+  host:
+    - llvm-openmp        # [osx]
+    - libgomp            # [linux64]
+    - openblas
+    - eigen
+    - bzip2
+    - xz
+  run:
+    - zlib
+    - bzip2
+    - xz
+    - libopenblas
+    - libgfortran-ng        # [linux64]
```

Comment:
This indicates that you are missing a fortran compiler.

Reply:
fortran actually not needed, this was copied from another blas recipe, can remove these libraries